docs/CEAP_en_US.md

```diff
@@ -0,0 +1,100 @@
+Article 1 - It can use, above the calculated quota for it's State, an extra of:
+1. BRL 1353,04, the Congress Person who's:
+  * Leader of a political party, parliament block, Minority, or leader of the Government in the Chamber of Deputies or the National Congress.
+  * Vice-leader of a political party or parliament block.
+  * President or vice-president of a Permanent Comission.
+  * Representative of political parties **with less members than a hundredth** of the total number of Congress People in the Chamber of Deputies.
+2. BRL 902,02, the Congress Person who's:
```

Comment:
* 902.02 (dot, not comma)
* It's **congressperson** (one word, no capital letter)
* I suggest _who is_ instead of _who's_